CHANGELOG.rst

```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+django-fsm 2.6.1 2018-12-14
```

Comment:
Forgot to update version to `2.7.0` here

Reply:
I've added the 2.7.0 part in the changelog.